pyat/at/lattice/lattice_object.py

```diff
@@ -170,7 +226,8 @@ def checkdipole(elem):
             n = mod_elem(self, checktype(elements.Quadrupole), quadrupole_func)
             print('{0} modified quadrupoles'.format(n))
```

Comment:
Can we please either remove these print statements or have some way of suppressing them as they are being very frequently spammed in ATIP.

Reply:
I can remove those prints!